docs/getting_oss_rules_of_play_rating.md

```diff
@@ -113,28 +118,30 @@ If everything is correctly set up, the output is going to look like the followin
-[+]       Does README mention REUSE?.............................Yes
-[+]       Does it have LICENSES directory with licenses?.........Yes
-[+]       Does it have a README file?............................Yes
-[+]       Does it have a contributing guideline?.................Yes
-[+]       Does it have a license file?...........................Yes
-[+]       Does it have a security policy?........................Yes
-[+]       Does it have a team with push privileges on GitHub?....Yes
-[+]       Does it have an admin team on GitHub?..................Yes
-[+]       Does it have enough admins on GitHub?..................Yes
-[+]       Does it have enough teams on GitHub?...................Yes
-[+]       Does it have unresolved vulnerability alerts?..........No
-[+]       Does it use an allowed license?........................Yes
-[+]       Does teams have enough members on GitHub?..............Yes
-[+]       Does the contributing guideline have required text?....Yes
-[+]       Does the license have disallowed content?..............No
-[+]       If a project's README doesn't contain required info:...Yes
-[+]       Is it compliant with REUSE rules?......................Yes
-[+]       Is it registered in REUSE?.............................Yes
-[+]   Explanation:..Found 1 recommendations
+[+]       Are vulnerability alerts enabled?.........................Yes
+[+]       Does README mention REUSE?................................Yes
+[+]       Does it have LICENSES directory with licenses?............Yes
+[+]       Does it have a README file?...............................Yes
+[+]       Does it have a code of conduct guideline?.................No
+[+]       Does it have a contributing guideline?....................Yes
+[+]       Does it have a license file?..............................Yes
+[+]       Does it have a security policy?...........................Yes
+[+]       Does it have a team with push privileges on GitHub?.......Yes
+[+]       Does it have an admin team on GitHub?.....................Yes
+[+]       Does it have enough admins on GitHub?.....................Yes
+[+]       Does it have enough teams on GitHub?......................Yes
+[+]       Does it have unresolved vulnerability alerts?.............No
+[+]       Does it use an allowed license?...........................No
+[+]       Does teams have enough members on GitHub?.................Yes
+[+]       Does the code of conduct guideline have required text?....No
+[+]       Does the contributing guideline have required text?.......Yes
+[+]       Does the license have disallowed content?.................No
+[+]       Is README incomplete?.....................................No
+[+]       Is it compliant with REUSE rules?.........................Yes
+[+]       Is it registered in REUSE?................................Yes
+[+]   Explanation:..Found 1 violated rule
 [+]
-[+] Rating:     9.0  out of 10.0 -> PASSED_WITH_WARNING
+[+] Rating:     0,0  out of 10,0 -> FAILED
```

Comment:
This output is not good. The rating value does not look good with 0. Also, it doesn't look good that it returns plain 0. Something maybe wrong here, while calculating the score. Could you check this?

Reply:
Same as above the same thing is happening on the master branch code as well for multiple repositories.

Reply:
https://github.com/SAP/fosstars-rating-core/pull/693#discussion_r714559323